iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -203,19 +204,22 @@ Viewer.prototype._findOrCreateWrapperNode = function(domNode) {
         nodes = this._wrapNode(domNode);
         // Create a node set of current node and all absolutely positioned
         // descendants.
-        nodes = nodes.find("*").addBack().filter(function () {
-            return (this == nodes[0] || $(this).css("position") == "absolute");
+        nodes = nodes.find("*").addBack().filter(function (n, e) {
+            // node list will include ix:* elements, with no style property.
+            // Can't trivially use instanceof HTMLElement in an iframe.
```

Comment:
I'm not very familiar with jQuery usage here, so I'm not following the comment.  What does "with no style property" mean?  Why would you want to use `instanceof`, and what doesn't work?

Reply:
> What does "with no style property" mean

`this.style.foo` will fail, because `this.style` is undefined.

> Why would you want to use instanceof, and what doesn't work?

I wanted to use `this instanceof HTMLElement` as the guard, but that is never true on elements inside an iframe – you need to use `iframe.contentWindow.HTMLelement`, and we don't have convenient access to the relevant iframe object here.

Reply:
Ah, so is the comment is justifying `'style' in this`?  That makes sense.  If you like, you could consider something like:
> The node list can contain non-HTMLElement ix:* nodes, so be defensive when checking for style.  In theory, we could use "instanceof HTMLElement" instead of "in", but that doesn't trivially work from an iframe.

Reply:
> this.style.foo will fail, because this.style is undefined.

You could use [optional chaining](https://caniuse.com/mdn-javascript_operators_optional_chaining) instead `this.style?.foo` as long as that's enough browser support.

Reply:
Yes, although it turns out that `getComputedStyle()` is less fussy - it only requires an `Element` as input, not an `HTMLElement`, so it doesn't object to the iXBRL nodes.